_posts/2022-04-11-nwaku-update.md

```diff
@@ -0,0 +1,277 @@
+The `nwaku` client now contains a working demonstration and integration of RLN relay.
+Check out [this tutorial](https://github.com/status-im/nim-waku/blob/ee96705c7fbe4063b780ac43b7edee2f6c4e351b/docs/tutorial/rln-chat2-live-testnet.md) to see the protocol in action using a toy chat application built on `nwaku`.
+
+## Future work
+
+As we continue working towards our goal of a fully decentralized, generalized and censorship-resistant messaging protocol,
+these are some of the current and future focus areas for `nwaku`:
+
+### Reaching out to operators:
+
+We want to push for operators to run and maintain their own Waku v2 nodes,
+preferably contributing to the default Waku v2 network as described by the default pubsub topic (`/waku/2/default-waku/proto`).
+Amongst other things, a large fleet of stable operator-run Waku v2 nodes will help secure the network,
+provide valuable services to a variety of applications
+and ensure the future sustainability of both Vac as a research organization and the Waku suite of protocols.
+
+We are targeting `nwaku` as the main option for operator-run nodes.  
+Specifically, we aim to provide through `nwaku`:
+
+1.  a lightweight and robust Waku v2 client.
+This client must be first in line to support innovative and new Waku v2 protocols,
+but configurable enough to serve the adaptive needs of various operators.
+2.  an easy-to-follow guide for operators to configure,
+set up and maintain their own nodes
+3.  a set of operator-focused tools to monitor and maintain a running node
+
+### Better conversational security layer guarantees
+
+Conversational security guarantees in Waku v2 are currently designed around the Status application.
+Developers building their own applications on top of Waku would therefore
+either have to reimplement a set of tools similar to Status
+or build their own security solutions on the application layer above Waku.
+We are working on [a set of features](https://github.com/vacp2p/research/issues/97) built into Waku
+that will provide the general security properties Waku users may desire
+and do so in a modern and simple way.
+As a first step, we've already made good progress toward [integrating noise handshakes](https://forum.vac.dev/t/noise-handshakes-as-key-exchange-mechanism-for-waku2/130) as a key exchange mechanism in Waku v2.
+
+### Protocol incentivization
+
+We want to design incentivization around our protocols to encourage desired behaviors in the Waku network,
+rewarding nodes providing costly services
+and punishing adversarial actions.
+This will increase the overall security of the network
+and encourage operators to run their own Waku nodes.
+In turn, the sustainability of Vac as an organization will be better guaranteed.
+As such, protocol incentivization was one of the major conclusions in our recent [Vac Sustainability and Business Workshop](https://forum.vac.dev/t/vac-sustainability-and-business-workshop/).
```

Comment:
replace conclusion with "focus" (makes it seem like it wasn't important before otherwise)